code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -224,18 +227,35 @@
 	if(!length(querys))
 		return
 	var/list/datum/SDQL2_query/running = list()
+	var/list/datum/SDQL2_query/waiting_queue = list() //Sequential queries queue.
+	
 	for(var/list/query_tree in querys)
 		var/datum/SDQL2_query/query = new /datum/SDQL2_query(query_tree)
 		if(QDELETED(query))
 			continue
 		if(usr)
 			query.show_next_to_key = usr.ckey
+		waiting_queue += query
+		if(query.options && SDQL2_OPTION_SEQUENTIAL)
```

Comment:
`&&` boolean operator used not `&` bitwise, probably will cause unintended behavior, like all queries being sequential if there are any options used

Reply:
Why no one reviews my PR's before merging.